aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
Ok so I checked Raymer again, and those constants all come from statistical analysis and best fit curves.  So, its seems like they don't have any meaning.